apps/freelance/src/redux/auth/auth-api.ts

```diff
@@ -0,0 +1,33 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const authApi = createApi({
+  reducerPath: 'authApi',
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:5000/auth/' }),
```

Comment:
today you have localhos, tomorrow https://mysuperwebsite/ you will all the time change URL?
Please read what it env variables are, why we need to use it, and when. Create the list for me: in which case you will use env, in which constants, and when you will write something directly in the code, and send it to me in private message, without this work I will not accept your PR)

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/87652792/201671121-90eecd38-4153-4ec8-8651-a661389de3a8.png)